### ISSUE TITLE
Unable to build polished on Windows

### ISSUE THREAD

**Issue by jbarco** [styled-components/polished#250]
Hi,
I forked the project and build it on a Mac successfully (apart from some errors regarding the document generation). However, if I try to build it on a Windows machine, it crashes at the next command:

`tsgen "lib/**/*.js.flow" --ignore "lib/**/_*.js.flow"`

- OS: **Windows 7**
- NodeJs version: **8.6.0**
- npm version: **5.3.0**

```
tsgen "lib/**/*.js.flow" --ignore "lib/**/_*.js.flow"

C:\polished\node_modules\tsgen\lib\walkers\index.js:26
            throw ctx.getError('Unsupported node type', node);
            ^

Error: Unsupported node type

import adjustHue from '../adjustHue'
  3 | 
  4 | describe('adjustHue', () => {
  5 |   it('should adjustHue of a reduced hex color', () => {
  6 |     expect(adjustHue(20, '#448')).toMatchSnapshot()
  7 |   })
    at Context.getError (C:\polished\node_modules\tsgen\lib\Context.js:63:16)
    at Object.walk [as default] (C:\polished\node_modules\tsgen\lib\walkers\index.js:26:23)
    at C:\polished\node_modules\tsgen\lib\index.js:29:76
    at Array.map (<anonymous>)
    at convertFile (C:\polished\node_modules\tsgen\lib\index.js:29:22)
    at C:\polished\node_modules\tsgen\lib\index.js:42:26
    at Array.forEach (<anonymous>)
    at Object.convertDirectory (C:\polished\node_modules\tsgen\lib\index.js:40:34)
    at Object.<anonymous> (C:\polished\node_modules\tsgen\lib\cli.js:16:4)
    at Module._compile (module.js:624:30)

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! polished@ build:typescript: `tsgen "lib/**/*.js.flow" --ignore "lib/**/_*.js.flow"`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the polished@ build:typescript script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
```

**Comment by bhough** [comment]
@ForbesLindesay any ideas on this one?

**Comment by ForbesLindesay** [comment]
Not sure, and I'm afraid I don't have easy access to a windows build to test it on.  If you narrow it down to something in tsgen I'd love a pull request :)

**Comment by bhough** [comment]
Addressed in 2.0: https://github.com/styled-components/polished/milestone/1
